Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -177,22 +176,30 @@
 end
 
 %% Define number of sessions to use

```

Comment:
Which sessions may be the technically correct case, however in the following update it will determine the number of sessions per subject individually so it would be change back. I suggest leaving it like this.